lib/sequenceserver/routes.rb

```diff
@@ -132,12 +132,19 @@ class Routes < Sinatra::Base
       erb :'400', :layout => nil, :locals => { :error => error }
     end
 
+    # This will catch error occurred during executing blast or blast_formatter
```

Comment:
How about creating a helper method, that takes the HTTP status to return and the template to render and update all error handlers to use this method?